transports/jetty/src/test/resources/jetty-address-binding-test.xml

```diff
@@ -1,40 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<mule xmlns="http://www.mulesoft.org/schema/mule/core"
-      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      xmlns:jetty="http://www.mulesoft.org/schema/mule/jetty"
-      xmlns:test="http://www.mulesoft.org/schema/mule/test"
-      xmlns:http="http://www.mulesoft.org/schema/mule/http"
-      xsi:schemaLocation="
-      http://www.mulesoft.org/schema/mule/jetty http://www.mulesoft.org/schema/mule/jetty/current/mule-jetty.xsd
-      http://www.mulesoft.org/schema/mule/test http://www.mulesoft.org/schema/mule/test/current/mule-test.xsd
-      http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
-      http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
-
-    <configuration>
-        <http:config useTransportForUris="true"/>
```

Comment:
unrelated to this issue, but we should make sure to have issues to remove all the backwards compatibility system properties